### DISCUSSION TITLE
猴痘本土案例漸增

### DISCUSSION THREAD

**Discussion by curtis1017** [1101-GS4719-C/Python-Community-GS4719-C#1238] [General]
猴痘確診病例截至4月6日，全台已累積23個案例，甚至從台北到高雄皆有，特別的是，這些人沒有出國旅遊史，所以可以得知猴痘的傳染鏈已偷偷在台灣默默形成。猴痘（monkeypox）最早在1958年被人類發現，主要常見於非洲嚙齒類動物身上，例如非洲松鼠、樹松鼠、睡鼠等。由於猴痘屬於人畜共通傳染病，去（2022）年5月在英國爆發以來，至今已傳遍全球上百個國家、數萬名案例。猴痘動物傳人的案例當中，確診者可能是因接觸染病動物的血液、體液、皮膚而感染，接著病毒會經由皮膚傷口、呼吸道、眼、鼻或口腔黏膜進入人體，潛伏期可從3到21天，平均約為6至13天。

再來我們講到患者感染猴痘初期，可能會出現發燒、畏寒、頭痛、肌肉痠痛、背痛、關節痛、淋巴腺腫大等症狀；之後皮疹會從臉部開始出現，並且逐漸延伸到身體各部位，最常見於手掌、足底處，也可能出現在生殖器、眼內。皮膚病灶顯現後，會演變成水泡，再逐漸結痂、脫落。由於皮疹通常會讓人感覺很癢，若是傷口沒有小心照顧或亂抓，便有可能留下疤痕。

如果害怕的人，可以盡快接種疫苗我國現行施打的猴痘疫苗為「JYNNEOS」，已在美國、加拿大、歐盟等國取得上市許可，預計施打兩劑、間隔至少四周。猴痘疫苗第二階段意願登記自4月10日下午兩點開放至5月12日，民眾先自我風險評估「近6個月內曾有高風險性行為」，符合條件者即可於上述期間的周一至周五，早上九點至下午五點，在意願登記平台進行登記，全台共有50家醫療院所可施打。

**Comment by sjh501** [comment]
目前的猴痘似乎還沒有特效藥，只能靠輸液治療或營養維持等治療。雖然在大約 2-3 周內便會結痂消退，且致死率並不高，但若免疫力低下還是有可能併發重症。如果要避免感染，要盡量避免與猴痘感染者或高風險者有密切接觸，另外前往疫區時避免接觸齧齒類和靈長類，如此便可大幅降低感染風險。
